src/partials/languages/nn.html

```diff
@@ -23,22 +23,43 @@
 <!-- Translations: Norwegian Nynorsk -->
 {% macro t(key) %}{{ {
   "language": "nn",
+  "direction": "ltr",
+  "announce.dismiss": "Ikkje vis dette att",
   "clipboard.copy": "Kopier til utklippstavla",
   "clipboard.copied": "Kopiert til utklippstavla",
+  "consent.accept": "Akseptert",
+  "consent.manage": "Innstillinger",
+  "consent.reject": "Reject",
   "edit.link.title": "Rediger denne sida",
   "footer.previous": "Førre",
   "footer.next": "Neste",
+  "footer.title": "Footer",
+  "header.title": "Header",
   "meta.comments": "Kommentarar",
   "meta.source": "Kjelde",
-  "search.config.lang": "no",
+  "nav.title": "Navigasjon",
+  "search.config.lang": "nn",
```

Comment:
Please remove the changes for `search.config.*` (here and in the other translation). I'm pretty confident that with those lines present, search will not work. [Lunr languages](https://github.com/MihaiValentin/lunr-languages#lunr-languages-----) only supports `no`.

Reply:
Much appriciated feedback @squidfunk! Sorry for not reading the **entire** contribution guide, my bad 🙄 

I notice in [lunr.no.js](https://github.com/MihaiValentin/lunr-languages/blob/master/lunr.no.js) that it mixes _both_ languages (`nn and `nb`), so I think the best to define `"search.config.lang": "no"`. Then the search will work too.